### DISCUSSION TITLE
Tutorial: How to create a physics enabled game

### DISCUSSION THREAD

**Discussion by CodeAndWeb** [axmolengine/axmol#1062] [Show and tell]
<a href="https://www.codeandweb.com/physicseditor/tutorials/how-to-create-a-physics-enabled-game-with-axmol-game-engine">
<img src="https://user-images.githubusercontent.com/716009/219702050-b9dacbfe-934e-49d9-ab1f-6bd0fcc8cff7.png"/>
</a>

[This tutorial](https://www.codeandweb.com/physicseditor/tutorials/how-to-create-a-physics-enabled-game-with-axmol-game-engine) explains, how to enable the 2d physics engine in Axmol. 

You learn how to:
- Enable physics in Axmol
- Create collision shapes the easy way
- Create an interactive game scene with falling objects

Full source code included.

[How to create a physics enabled game with Axmol Game Engine](https://www.codeandweb.com/physicseditor/tutorials/how-to-create-a-physics-enabled-game-with-axmol-game-engine)

**Comment by johnspeny** [comment]
Thanks for this update and info,  I created physics bodies using P.E, i prefer to use box2d. Could you please update the parser for it to work with axmole. The last time i checked it was too old

**Comment by gradyplayer** [comment]
@CodeAndWeb:
I just went through this tutorial, it is good but the CMake stuff doesn't work with latest axmol... there is a call to `setup_ax_app_config` that isn't defined.

the other thing is a new axmol project will create multitouch versions of the touch callbacks which will foul up std::bind in `listener->onTouchBegan = AX_CALLBACK_2(HelloWorld::onTouchesBegan, this);` 

Probably won't confuse anyone as much as it did me; but could be worth updating.

It would be cool to start from the start with the images and TexturePacker, I know it would be longer, but probably give a better indication of how those work together (since I was remaking the project it wasn't clear that you had to have the sprite sheet plist already exported to export the physics to).
